sklearn/utils/tests/test_utils.py

```diff
@@ -199,11 +202,35 @@ def test_safe_indexing():
     assert_array_equal(np.array(X_inds), np.array(X)[inds])
 
 
+@pytest.mark.parametrize("idx", [0, [0, 1]], ids=['scalar', 'list'])
+def test_safe_indexing_axis_1_sparse(idx):
+    X_true = safe_indexing(X, idx, axis=1)
+
+    X_sparse = sp.csc_matrix(X)
+    assert_array_equal(
+        safe_indexing(X_sparse.toarray(), idx, axis=1), X_true
```

Comment:
I don't understand this test. Why are you converting the sparse matrix to an array?

Reply:
I am checking that indexing a sparse array will give the same than a dense one

Reply:
You are converting the sparse array to a dense array

Reply:
Oh ups I see. I wanted to convert the output. Good catch